pallets/scoring/Cargo.toml

```
@@ -0,0 +1,66 @@
+[package]
+name = "pallet-scoring"
+description = "Manage ONDC Netwrok Scores."
```

Lets not 'restrict' this to ONDC Network.

```suggestion
description = "Manage Score of entities."
```

---

or just say "Manage network scores"